modules/config/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+##################################################################
+#
+# cFE Executive Services (ES) module CMake build recipe
```

Comment:
"Config module CMake build recipe"

Reply:
Ahh yes, cut and paste evidence I forgot to fix.

Also worth noting that the coverage / functional test aspects are not at all implemented yet, waiting on getting some feedback before going that far....

Reply:
Yeah I noticed it is still a draft so didn't really review the code

Reply:
cut/paste error fixed in 145cecb5